### ISSUE TITLE
OpCapability Kernel missing in a few cases

### ISSUE THREAD

**Issue by pavanky** [KhronosGroup/SPIRV-LLVM#145]
code to reproduce:

```
float negate(float in)
{
    return -in;
}
```

Compilation commands:

```
CLANG=$WORK_DIR/local/bin/clang++
LLVM_SPIRV=$WORK_DIR/local/bin/llvm-spirv

$CLANG -cc1 -emit-llvm-bc -triple spir64-unknown-unknown -include $SPIR_INC_FILE -o $FILE_NAME.bc $FILE_NAME.cl
$LLVM_SPIRV $FILE_NAME.bc -o $FILE_NAME.spv64
```

Error caught running it through the `spirv-val` validator

**Comment by bader** [comment]
Interesting.
If I use -emit-spirv option to get SPIR-V file from clang I get all capabilities right.
I guess llvm-spirv tool incorrectly handles OpenCL code w/o kernels.

**Comment by bader** [comment]
Fixed by #156.
